sfdo_template_helpers/admin/middleware.py

```diff
@@ -16,8 +16,15 @@ def __init__(self, get_response):
         self.ip_ranges = settings.ADMIN_API_ALLOWED_SUBNETS
 
     def __call__(self, request):
-        if request.path.startswith(f"/{settings.ADMIN_AREA_PREFIX}"):
-            self._validate_ip(request)
+        restricted_areas = tuple(getattr(settings, "RESTRICTED_PREFIXES", ()))
+        admin_area = getattr(settings, "ADMIN_AREA_PREFIX", None)
+        if admin_area is not None:
+            restricted_areas += (admin_area,)
```

Comment:
You can get rid of this conditional if you return an empty tuple instead of `None` as the default for the `getattr` call above.